jaseci_ai_kit/jac_nlp/jac_nlp/bi_enc/bi_enc.py

```diff
@@ -35,17 +35,27 @@ def setup():
     Loading configurations from utils/config.cfg and
     initialize tokenizer and model
     """
-    global model, tokenizer, model_config, train_config, t_config_fname, m_config_fname
+    global model, tokenizer, model_config, train_config, t_config_fname, m_config_fname, BI_ENC_ROOT  # noqa
```

Comment:
Let's have `BI_ENC_ROOT` at the top level of this file/module. Its not really a global variable. It is a constant.

Reply:
Same with the other modules too.